testsuite/forge-cli/src/main.rs

```diff
@@ -208,19 +210,26 @@ fn main() -> Result<()> {
         CliCommand::Test(ref test_cmd) => {
             // Identify the test suite to run
             let mut test_suite = get_test_suite(args.suite.as_ref())?;
+
+            // Identify the number of validators and fullnodes to run
             if let Some(num_validators) = args.num_validators {
-                match NonZeroUsize::new(num_validators) {
-                    Some(num_validators) => {
-                        test_suite = test_suite.with_initial_validator_count(num_validators)
-                    }
-                    None => {
+                let num_validators_non_zero =
+                    NonZeroUsize::new(num_validators).expect("--num-validators must be positive!");
```

Comment:
`.context()?` instead, since this function returns a result?

Reply:
Done.